ethcore/src/miner/transaction_queue.rs

```diff
@@ -488,6 +495,11 @@ pub enum PrioritizationStrategy {
 	GasFactorAndGasPrice,
 }
 
+/// Point in time when transaction was inserted.
+/// (implying block numbers)
+pub type InsertionTime = u64;
```

Comment:
If this is blocks, might want to change type to `BlockNumber`, and name to `QueuingPeriod`.